keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1656,30 +1662,26 @@ gets updated now if the field doesn't
 match the current list already. If no PodScheduling object exists yet,
 it gets created.
 
-When there is a scheduler extender configured for the claim, creating and
-updating PodScheduling objects gets skipped because the scheduler
-extender handles filtering.
-
 #### Reserve
 
 A node has been chosen for the Pod.
 
 If using delayed allocation and the resource has not been allocated yet,
-the SelectedNode field of the ResourceClaim
+the PodSchedulingSpec.SelectedNode field
 gets set here and the scheduling attempt gets stopped for now. It will be
-retried when the ResourceClaim status changes.
+retried when the ResourceClaim or PodScheduling statuses change.
 
 If all resources have been allocated already,
```

Comment:
Wasn't it the driver that added `reservedFor` when doing the allocation?

Reply:
Yes, but only as an optimization for delayed allocation. The scheduler still needs to do it for claims with immediate allocation.

Reply:
Updated. I hope it is clearer now.